src/bluetooth.py

```diff
@@ -0,0 +1,30 @@
+import bluetooth
```

Comment:
Where is this bluetooth coming from? If you require special packages, you should create a requirements.txt to let us know what packages we need to install. Same with pexpect if that's not built in.

Reply:
I'll get on that right now.